js/src/models/shortcode.js

```diff
@@ -87,6 +87,11 @@ Shortcode = Backbone.Model.extend({
 			content = this.get( 'inner_content_backup' );
 		}
 
+		//Run autop before shooting into view
+		if( content ) {
+			content = window.wp.editor.autop( content );
```

Comment:
Elsewhere we're already doing something similar: https://github.com/fusioneng/Shortcake/blob/master/js/src/utils/shortcode-view-constructor.js#L231

Basically creating a custom unAutoP method that internally calls the core WP functionality. It allows us to change this more easily in the future.

Actually - the existing code uses the unAutoP functionality in `switchEditors` I wonder whether we should use this for autoP too. Not sure if the functionliaty is different.